docs/core/project-sdk/msbuild-props.md

```diff
@@ -12,6 +12,123 @@ This page is a reference for the MSBuild properties and items that you can use t
+- [ApiCompatValidateAssembliesSemaphoreFile](#apicompatvalidateassembliessemaphorefile)
+- [NoWarn](#nowarn)
+- [RoslynAssembliesPath](#roslynassembliespath)
+
+### ApiCompatLeftAssemblies
+
+The `ApiCompatLeftAssemblies` item specifies the path to one or more assemblies that serve as the *left side* to compare. This property is required when `ApiCompatValidateAssemblies` is set to `true`.
+
+```xml
+<ItemGroup>
+  <ApiCompatLeftAssemblies Include="path/to/left-assembly" />
+</ItemGroup>
+```
+
+### ApiCompatLeftAssembliesReferences
+
+The `ApiCompatLeftAssembliesReferences` item specifies the path to assembly references or the underlying directories for the left side of the comparison.
+
+```xml
+<ItemGroup>
+  <ApiCompatLeftAssembliesReferences Include="path/to/assembly-references" />
+</ItemGroup>
+```
+
+### ApiCompatLeftAssembliesTransformationPattern
+
+The `ApiCompatLeftAssembliesTransformationPattern` item specifies a transformation pattern for the left side assemblies.
+
+```xml
+<ItemGroup>
+  <ApiCompatLeftAssembliesTransformationPattern Include="$(_ApiCompatCaptureGroupPattern)" ReplacementString="ref/$1/$2" />
+</ItemGroup>
+```
+
+### ApiCompatRightAssemblies
+
+The `ApiCompatRightAssemblies` item specifies the path to one or more assemblies that serve as the *right side* to compare. This property is required when `ApiCompatValidateAssemblies` is set to `true`.
+
+```xml
+<ItemGroup>
+  <ApiCompatRightAssemblies Include="path/to/right-assembly" />
+</ItemGroup>
+```
+
+### ApiCompatRightAssembliesReferences
+
+The `ApiCompatRightAssembliesReferences` item specifies the path to assembly references or the underlying directories for the right side of the comparison.
+
+```xml
+<ItemGroup>
+  <ApiCompatRightAssembliesReferences Include="path/to/assembly-references" />
+</ItemGroup>
+```
+
+### ApiCompatRightAssembliesTransformationPattern
+
+The `ApiCompatLeftAssembliesTransformationPattern` item specifies a transformation pattern for the right side assemblies.
+
+```xml
+<ItemGroup>
+  <ApiCompatRightAssembliesTransformationPattern Include="$(_ApiCompatCaptureGroupPattern)" ReplacementString="lib/$1/$2" />
+</ItemGroup>
+```
+
+### ApiCompatStrictMode
+
+When set to `true`, the `ApiCompatStrictMode` property specifies that API compatibility checks should be performed in strict mode.
+
+```xml
+<PropertyGroup>
+  <ApiCompatStrictMode>true</ApiCompatStrictMode>
+</PropertyGroup>
+```
+
+### ApiCompatValidateAssemblies
+
+The `ApiCompatValidateAssemblies` property enables a series of validations on the specified assemblies. For more information, see [Assembly validation](../../fundamentals/apicompat/assembly-validation.md).
+
+```xml
+<PropertyGroup>
+  <ApiCompatValidateAssemblies>true</ApiCompatValidateAssemblies>
+</PropertyGroup>
+```
+
+### ApiCompatValidateAssembliesSemaphoreFile
+
+The `ApiCompatValidateAssembliesSemaphoreFile` property specifies the path to the *.semaphore* file for the assembly validation.
```

Comment:
Not sure what this property is for or if I should document it.

Reply:
Nope that shouldn't be documented.

Reply:
https://github.com/dotnet/sdk/pull/37276